stellargraph/layer/gcn_lstm.py

```diff
@@ -187,7 +187,7 @@ def call(self, features):
         return output
 
 
-@experimental(reason="Lack of unit tests and some code refinement", issues=[1131, 1132])
+# @experimental(reason="Lack of unit tests and some code refinement", issues=[1131, 1132])
```

Comment:
I think this PR doesn't fully take this out of experimental. I think your other PR is needed, plus maybe some polish work (in another PR) to use  to use `kernel_constraint` in the layers etc.

Reply:
I think we should also change how the model is constructed and used, to be more similar to the other models here: #1526, #1564. This is likely to require significant changes to the interface and thus would be best while it's still `@experimental`.